annotator/src/test/java/org/cbioportal/annotator/mixin/TranscriptConsequenceMixin.java

```diff
@@ -0,0 +1,68 @@
+
+package org.cbioportal.annotator.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TranscriptConsequenceMixin {
```

Comment:
This mixin is only needed within the test scope. All other mappings are handled by the auto generated API client.

Reply:
@onursumer can't we import this class from genome-nexus?

Reply:
We have something similar in genome nexus, but not the same mixin. We are using this mixin to map mock data onto an instance in tests. So, not sure if it is worth importing from Genome Nexus.

Reply:
👌